src/Core/src/Handlers/SearchBar/SearchBarHandler.Windows.cs

```diff
@@ -159,6 +159,7 @@ void OnLoaded(object sender, UI.Xaml.RoutedEventArgs e)
 			if (VirtualView != null)
 			{
 				PlatformView?.UpdateTextColor(VirtualView, _defaultTextColorBrush, _defaultTextColorFocusBrush, _queryTextBox);
+				PlatformView?.UpdatePlaceholderColor(VirtualView, _defaultPlaceholderColorBrush, _defaultPlaceholderColorFocusBrush, _queryTextBox);
```

Comment:
The placeholder color was not working as expected. I have added in this PR this change also to fix the issue.